src/components/product-tile/ProductTile.tsx

```diff
@@ -50,6 +52,10 @@ const ProductTileContainer = styled.li`
 
 export const ProductTile: React.FC<{ product: Product }> = ({ product }) => {
   // TODO: Add translations and currency formatting
+  const { t } = useTranslation();
+  const changeLanguage = (lang: string) => {
```

Comment:
Instead of typing it as a string, you can type it has the acceptable strings. You can create a type: supportedLanguagesType like "en" | "pt" etc.

Reply:
Instead of changing the language inside the product tile, you could do it in the header. Near the hamburger button

Reply:
got it, ill make all the changes!